staging/src/k8s.io/apiserver/pkg/registry/rest/rest.go

```diff
@@ -63,6 +63,16 @@ type Storage interface {
 	Destroy()
 }
 
+// StorageWithReadiness extends Storage interface with the readiness check.
+//
+// FIXME: Should we instead change the Storage interface above?
```

Comment:
@jpbetz - thoughts?

Reply:
I think it's slightly better the way it is. Biasing toward backward compatibility is generally what we want.  Maybe add a comment to the Storage interface recommending users consider using StorageWithReadiness if there is an appropriate readiness check to include?

Reply:
done